tests/unit/amo/components/TestVPNPromoBanner.js

```diff
@@ -268,15 +275,15 @@ describe(__filename, () => {
       });
       const _tracking = createFakeTracking();
       render({ _tracking, _localStorage });
+      _tracking.sendEvent.mockClear();
```

Comment:
This is only needed with the current versions of the dependencies, we're getting an extra impression event, presumably because of the forced `render()` + `history.push`...

Reply:
I'm not sure why this would still be the case. I'm going to see if I can do some debugging with your patch to see if I can figure out what's going wrong. I just want to make sure it isn't a side effect that's going to potentially cause problems for us elsewhere.

Reply:
I tested with your branch, rebased on top of the dependency updates, and if I remove both changes in these tests (the `mockClear` and the change in the number of expected calls), the tests pass. Is that not what you were seeing?